content/_index.md

```diff
@@ -29,12 +29,24 @@ functionality like the UEFI DXE phase with a Linux kernel and runtime.
   [here](http://slack.u-root.com))
 * [Twitter](https://twitter.com/LinuxBootOrg)
 
-### More info?
+### Getting started:
 
-* [Trammell Hudson's LinuxBoot talk at 34c3](https://trmm.net/LinuxBoot_34c3)
-* Ron Minnich's talk "Replace your exploit-ridden firmware with a Linux kernel": [video](https://www.youtube.com/watch?v=iffTJ1vPCSo) / [slides](https://schd.ws/hosted_files/osseu17/84/Replace%20UEFI%20with%20Linux.pdf)
-* [Heads, the slightly more secure firmware](http://osresearch.net/)
-* [NERF branch of Heads for building LinuxBoot ROMs](https://github.com/osresearch/heads/tree/nerf)
+```
+git clone https://github.com/osresearch/heads
+make -C heads BOARD=qemu-linuxboot all run
+```
+
+### In the news:
+
+* [LWN.net: LinuxBoot: Linux as firmware](https://lwn.net/SubscriberLink/748586/b665913279cdbc1c/) (2018-03-08)
+* [Linux Journal: FOSS Project Spotlight: LinuxBoot](https://lwn.net/SubscriberLink/748586/b665913279cdbc1c/) (2018-02-15)
```

Comment:
do we want to give subscriber links here? They should become accessible in matter of days